cypress/integration/spawn/volume.ts

```diff
@@ -8,6 +8,7 @@ describe("Navigating to Spawn Volume page", () => {
 
   beforeEach(() => {
     cy.preserveCookies();
+    cy.viewport(1920, 1600);
```

Comment:
I dont think we should change the viewPort  size to accommodate tests. The features should work regardless of the browser dimensions